.github/workflows/production-build.yml

```diff
@@ -136,3 +136,40 @@ jobs:
           dot-npmrc: ${{ secrets.PUBLISHING_PAT }}
           working-directory: ${{ matrix.lib.dist }}
       - run: cd ${{ matrix.lib.dist }} && npm publish
+
+  hotfix-branch:
+    needs: bump-version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Github Action
+        uses: actions/checkout@master
+        with:
+          token: ${{ secrets.AUTOMATION_PAT }}
+          fetch-depth: 0
+      - name: Get Previous tag
+        id: previoustag
+        uses: 'WyriHaximus/github-action-get-previous-tag@v1'
+        with:
+          fallback: 0.0.0
+      - name: Create Branch
+        uses: peterjgrainger/action-create-branch@v2.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.AUTOMATION_PAT }}
+        with:
+          branch: 'hotfix/v${{ steps.previoustag.outputs.tag }}'
+
+  delete-old-hotfixes:
```

Comment:
We should only delete old hotfix branches if the release is more than a month old, to keep room for hotfixes if needed or any reverts needed

Reply:
I think a 2 month buffer that this uses should be plenty